packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -22,6 +22,7 @@ import ActionBar from "./action-bar";
 import DevTools, { FrameworkEnum } from "./dev-tools";
 import ConfigurationPanel from "./configuration-panel";
 import NotFound from "./not-found";
+import { convertStylePropertyPixelsToNumber } from "@microsoft/fast-web-utilities";
```

Comment:
this doesn't appear to get used

Reply:
@janechu This is being used on line 832 and 833